Cargo.toml

```diff
@@ -62,6 +62,7 @@ num-complex = "0.4"
 glob = "0.3"
 quickcheck = "1"
 criterion = "0.5.0"
+itertools = "0.13.0"
```

Comment:
Why is this needed?

Reply:
For the `itertools::assert_equal()` method in the iterators test.